src/xdocs/config_naming.xml

```diff
@@ -2172,15 +2172,38 @@ public boolean equals(Object o) {
         <source>
 &lt;module name=&quot;StaticVariableName&quot;/&gt;
         </source>
+        <p>Code Example:</p>
+        <div class="wrapper">
+          <pre>
+class MyClass {
+  public static int goodStatic = 2; // OK
+  private static int BadStatic = 2; // violation, name 'BadStatic'
+                                    // must match pattern '^[a-z][a-zA-Z0-9]*$'
+}
+          </pre>
+        </div>
         <p>
           An example of how to configure the check for names that begin with a lower case letter,
-          followed by letters, digits, and underscores is:
+          followed by letters, digits, and underscores. Also, suppress the check from being
+          applied to private type:
         </p>
         <source>
-&lt;module name="StaticVariableName"&gt;
-  &lt;property name="format" value="^[a-z](_?[a-zA-Z0-9]+)*$"/&gt;
+&lt;module name=&quot;StaticVariableName&quot;&gt;
+  &lt;property name=&quot;format&quot; value=&quot;^[a-z](_?[a-zA-Z0-9]+)*$&quot;/&gt;
+  &lt;property name=&quot;applyToPrivate&quot; value=&quot;false&quot;/&gt;
```

Comment:
we try to show examples for each property
https://checkstyle.org/config_naming.html#StaticVariableName_Properties

please extend example to use all `applyXxxxxxxx`

Reply:
I add a new example to show how to configure the check with public and protected type suppressed. Can help me check if this is ok?

Also, the wercker pipeline is failing, I am not sure how it works as the page show 401......Can tell me how to read this check?

Reply:
@0blivious unfortunately, wercker does not allow anonymous access like the other CIs so only the admins can view it.

It failed on `Sonarqube validation` but it isn't reporting an error and the direct sonar page says it passed. It looks to me like its a false error.

Reply:
From log of sonar build item:
`Sonar API Token has no access rights` 
But expected is `Sonar Qualitygate is OK` , something wrong with token.
This failure can be ignored.
I am restarting wercker.